src/content/about/marketing.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Marketing activities
+description: Accepted practices for corporate involvement in the nf-core community
+---
+
+This page is intended for anyone coming to the Nextflow and nf-core community when representing a company or corporate interest.
+
+Firstly - welcome! We are an open and inclusive community and encourage everyone to be involved. Here we try to set out some guidelines to act as guard-rails for activity, both for the benefit of community members but also so that you feel comfortable that you are interacting with the community in acceptable ways.
+
+## Code of conduct
+
+The [code of conduct](/code_of_conduct/) applies to everyone active in the community.
+Two points are particularly worth highlighting, which are not allowed:
+
+- Sending unsolicited direct-messages or emails to community members on Slack / other platforms
+- Posting in nf-core spaces specifically to criticise or detract from Nextflow and nf-core, and to promote alternatives
+
+Contravening these, or any of the behaviours listed in the code of conduct, will likely result in an immediate ban.
+
+## Slack channels
+
+Please refrain from posting about your products and services in Slack, with some exceptions:
+
+- The `#shameless-promo` channel
+  - This channel is designed specifically for promotion of personal and commercial projects relevant to the nf-core community
+  - "Relevant" means that a significant chunk of the people reading the message may find it useful for their daily work.
+  - See above code of conduct points. Posting that tool X is better than Nextflow / nf-core and advertising an alterative is not ok.
```

Comment:
not sure if X is a good name for a placeholder tool anymore 😉

Reply:
😆